bears/general/IndentationBear.py

```diff
@@ -66,6 +66,7 @@ def run(self,
             Full path of external directory containing the coalang
             file for language.
         """
+        ranges = {}
```

Comment:
i'm sure only the next commit(with keyword) would have deleted more lines, it is an improvement for sure since i dont have to pass around different ranges and also is helpful when we have to update these ranges when they change during the process